package.json

```diff
@@ -70,15 +70,20 @@
     "fix": "gts fix",
     "prepare": "npm run compile",
     "pretest": "npm run compile",
-    "posttest": "npm run check"
+    "posttest": "npm run check",
+    "postinstall": "cd test/winston-2 && npm install"
```

Comment:
Is this going to execute if users `npm install`?

Reply:
good catch =)